src/vite-env.d.ts

```diff
@@ -1 +1,4 @@
 /// <reference types="vite/client" />
+
+declare module '@agoric/web-components/react';
```

Comment:
@turadg This is something I wanted to ask about, if I don't add these it complains about missing a `.d.ts` file in the package. Is this the ambient types problem? Is there a better way around this? I'd like to get the types for these packages, because now it's just `any`.

Reply:
Thanks, I didn't notice this. Using [shorthand ambient modules](https://www.typescriptlang.org/docs/handbook/modules.html#shorthand-ambient-modules) is okay to get started, but yeah it makes everything `any`.

These SDK modules don't export types, unfortunately. Doing that has wide repercussions so let's punt that to post rc